index.html

```diff
@@ -0,0 +1,343 @@
+          <ul class="social-media-icons">
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/github.png" alt="git" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/Linkedin.png" alt="linkedIn" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/angelite.png" alt="image" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/twitter.png" alt="image" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/medium.png" alt="image" />
+              </a>
+            </li>
+          </ul>
+          <div class="new-code">
+            <button type="submit" class="go-newmes">My resume</button>
+          </div>
+          <ul class="skill">
+            <li class="event12">
+              <li class="event7">
+                <h3 class="languages">Languages</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
+                </button>
+              </li>
+              <li>
+                <ul class="language-list">
+                  <li class="JavaScript1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-1.png" alt="" />
+                      <p>JavaSript</p>
+                    </div>
+                  </li>
+                  <li class="HTML1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-2.png" alt="" />
+                      <p>HTML</p>
+                    </div>
+                  </li>
+                  <li class="CSS1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-3.png" alt="" />
+                      <p>CSS</p>
+                    </div>
+                  </li>
+                </ul>
+              </li>
+            </li>
+            <ul class="event-skill">
+              <li class="event18">
+                <h3>Frameworks</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Disabled.png" alt="" />
+                </button>
+              </li>
+              <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+              <li class="event18">
+                <h3>Skills</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Disabled.png" alt="" />
+                </button>
+              </li>
+              <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section id="consection">
```

Comment:
- [ ] Good job on completing the functionality of the form, however you have not completed the design part which have been pointed with the red circles and underline in the image below
- [ ] Your fonts and color do not match what is on figma for your `contact me title`
- [ ] Your paragraph as well does not match the figma provided design as shown below and it is not properly centered
- [ ] The form fields for `name & email` do not have the appropriate border radius and they are not properly centered including the textare
- [ ] Please improve on the styling for this section before resubmitting your project for a review :+1: :sparkles: 
- figma design
![fm](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/89055502/ff403553-9801-41bb-be9d-6fdf0818eea4)
- your design
![mf](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/89055502/9d9b7179-1754-41b0-a1f4-3e880cc2d880)

Reply:
**Good job on completing the functionality of the form, however you have not completed the design part which have been pointed with the red circles and underline in the image below**:
The red circle is not a part of the design. Maybe somebody is working in this point now:
![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/1eed961d-9020-40fb-8776-a9d97e7d6c8d)

**Your fonts and color do not match what is on figma for your contact me title**: Contact Me is fixed now. Actually, I am using a family Poppins similar, because Poppins alone contains problems, so I am using: (font-family: "Poppins", sans-serif;)
![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/2e4bfcc8-11ef-4184-ad30-06ea285f1790)

**Your paragraph as well does not match the figma provided design as shown below and it is not properly centered**:
All is fixed and in the center.
![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/5d1e3f0e-3ff9-4405-8adf-16192949d32b)

**The form fields for name & email do not have the appropriate border radius and they are not properly centered including the textare** Name & email fixed; Border radius fixed and in the center
![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/3bf122a4-e8b3-4b3e-b356-b61188aee9f9)

**Please improve on the styling for this section before resubmitting your project for a review 👍 ✨** As I mentioned before, I am using (font-family: "Poppins", sans-serif;) because Poppins alone cant use at this moment. It was recommended for another code reviwer.